ui/src/components/Dashboard/UserMenu.jsx

```diff
@@ -34,7 +33,7 @@ function UserMenu(props) {
   const handleCopyAccessToken = async () => {
     const accessToken = await auth0.getAccessTokenSilently();
 
-    copy(accessToken);
+    navigator.clipboard.writeText(accessToken);
```

Comment:
Is this missing `await`?

Reply:
It was missing one before and it's still missing it and yet it works... Gotta love this language